src/screens/login/Login.jsx

```diff
@@ -54,10 +57,13 @@ const Login = ({ navigation }) => {
         <Image source={Logo} style={styles.logo} />
       </View>
 
-      <Typography id="USERNAME" style={styles.textFontInput} />
+      <Typography
+        id="USERNAME"
+        style={[globalStyles.h2Text, globalStyles.inputHeader]}
```

Comment:
```suggestion
        style={globalStyles.h2Text, globalStyles.inputHeader}
```
The arrays have some conflicts with Typography component when it merges styles. Removing the array brackets seem to resolve issue